src/main/app/src/app/bag/bag-zaken-tabel/bag-zaken-tabel.component.html

```diff
@@ -5,13 +5,13 @@
 
 <mat-card>
   <mat-card-header>
-    <mat-card-title>{{'zaken'| translate}}</mat-card-title>
+    <mat-card-title>{{ "zaken" | translate }}</mat-card-title>
```

Comment:
Spotless, linting, IDEA issue? Is this not defined in our Spotless ESlint etc config? Makes reviewing this PR quite hard..

Reply:
Reverted the change; I need time to align Spotless and VSC setup

Reply:
Welll GRRR; my local spotlessAppApply did not do a nice job again; so looking very weird now

Reply:
Hmm.. `./gradlew spotlessApply` should do the same job for all of us, regardless of our local setup. That's the whole idea. If this does not work we have an issue.. @hsiliev any thoughts?

Reply:
Yep; but there is a bit of unalignment that, when saving with VSC, spotless is not picking up; in HTML files; so that needs  some attention. I don't want to switch back to IntelliJ.

Reply:
HTML files are now reverted BTW

Reply:
VSC plugins have some overrides for the formatters used by spotless (prettier), so this can happen indeed.

Would be good to investigate if we can disable those override mechanisms